src/utils/curlRequestParser.ts

```diff
@@ -52,7 +52,7 @@ export class CurlRequestParser implements RequestParser {
         }
 
         // parse body
-        let body = parsedArguments.d || parsedArguments.data || parsedArguments['data-ascii'] || parsedArguments['data-binary'] || parsedArguments['data-raw'];
+        let body = parsedArguments.d || parsedArguments.data || parsedArguments['data-ascii'] || parsedArguments['data-binary'] || parsedArguments['data-raw'] || parsedArguments['data-urlencode'];
```

Comment:
What about putting all arguments in array in order to scale or abstract if we need it? for example:

Reply:
```suggestion
        const dataKeys = ['d', 'data', 'data-ascii', 'data-binary', 'data-raw', 'data-urlencode'];
        let body = parsedArguments[dataKeys.find(key => !!parsedArguments[key])];
```